.circleci/config.yml

```diff
@@ -2,49 +2,93 @@
 #
 # Check https://circleci.com/docs/2.0/language-javascript/ for more details
 #
+
+aliases: 
+  - &environment
+      docker:
+        # specify the version you desire here
+        - image: circleci/node:8.9.0
+        
+        # Specify service dependencies here if necessary
+        # CircleCI maintains a library of pre-built images
+        # documented at https://circleci.com/docs/2.0/circleci-images/
+        # - image: circleci/mongo:3.4.4
+      working_directory: ~/Prebid.js
+
+  - &restore_dep_cache
+      keys:
+        - v1-dependencies-{{ checksum "package.json" }}
+        # fallback to using the latest cache if no exact match is found
+        - v1-dependencies-
+
+  - &save_dep_cache
+      paths:
+        - node_modules
+      key: v1-dependencies-{{ checksum "package.json" }}
+
+  - &install
+      name: Install gulp cli
+      command: sudo npm install -g gulp-cli
+
+  - &run_unit_test
+      name: BrowserStack testing
+      command: gulp test --browserstack --nolintfix
+
+  - &run_endtoend_test
+      name: BrowserStack End to end testing
+      command: echo "Running end to end test using workflows"
```

Comment:
Can you clarify how this command starts the e2e test process?  I don't see the normal gulp e2e command in this file and this command appears to just print the noted text.

Reply:
I used echo to verify workflow. Updated the command now